nibiru/client.py

```diff
@@ -72,6 +72,12 @@ def get_block_by_height(
         req = tendermint_query.GetBlockByHeightRequest(height=height)
         return self.stubCosmosTendermint.GetBlockByHeight(req)
 
+    def get_blocks_by_height(
+        self, start_height: int, end_height: int
+    ) -> Generator[tendermint_query.GetBlockByHeightResponse, None, None]:
+        for height in range(start_height, end_height):
```

Comment:
`end_height` can be an optional arg so that the generator goes up to the latest block if `end_height` isn't given.